### ISSUE TITLE
Rubberduck>Settings>Unit Test Settings>Binding Mode

### ISSUE THREAD

**Issue by IvenBach** [rubberduck-vba/Rubberduck#2967]
When changing Binding Mode from Early to Late is it possible to remove the reference to the Rubberduck.tlb automatically?

**Comment by retailcoder** [comment]
Absolutely... except we would first have to be sure no early-bound references to the RD API exist in the user's code.

**Comment by IvenBach** [comment]
That would require adding code that checks in the parser before doing so? If there's any bugs just have @Hosch250 fix them?

**Comment by retailcoder** [comment]
We'll review the PR, no worries :-)
